tests/integration/serde/timestamps.rs

```diff
@@ -0,0 +1,42 @@
+use serde::{Deserialize, Serialize};
+use serde_test::{assert_de_tokens_error, assert_tokens, Configure, Token};
+use time::macros::datetime;
+use time::serde::timestamp;
+use time::OffsetDateTime;
+
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
```

Comment:
I don't think we actually need the derives other than serde's.

Reply:
I can remove `Eq` but everything else is required by [`assert_tokens`](https://docs.rs/serde_test/1.0.130/serde_test/fn.assert_tokens.html)